.jenkins

```diff
@@ -191,7 +191,7 @@ pipeline {
                         sh '''rm -rf build && mkdir -p build && cd build && \
                               cmake \
                                 -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-                                -DCMAKE_CXX_COMPILER=clang++ \
+                                -DCMAKE_CXX_COMPILER=/opt/llvm/bin/clang++ \
```

Comment:
Was this change actually necessary?

Reply:
Not anymore. Resolved the `PATH` issue in the Dockerfile which fixes this issue.